x-pack/plugins/observability_solution/observability_ai_assistant_management/public/routes/components/knowledge_base_edit_manual_entry_flyout.tsx

```diff
@@ -45,18 +46,19 @@ export function KnowledgeBaseEditManualEntryFlyout({
   const { mutateAsync: deleteEntry, isLoading: isDeleting } = useDeleteKnowledgeBaseEntry();
 
   const [isPublic, setIsPublic] = useState(entry?.public ?? false);
-
-  const [newEntryId, setNewEntryId] = useState(entry?.id ?? '');
+  const [newEntryTitle, setNewEntryTitle] = useState(entry?.title ?? entry?.doc_id ?? '');
   const [newEntryText, setNewEntryText] = useState(entry?.text ?? '');
 
-  const isEntryIdInvalid = newEntryId.trim() === '';
+  const isEntryIdInvalid = newEntryTitle.trim() === '';
   const isEntryTextInvalid = newEntryText.trim() === '';
   const isFormInvalid = isEntryIdInvalid || isEntryTextInvalid;
 
   const handleSubmit = async () => {
     await createEntry({
       entry: {
-        id: newEntryId,
+        id: entry?.id ?? v4(),
+        doc_id: entry?.doc_id,
```

Comment:
`doc_id` is no longer editable by the end user. It's an identifier decided by the LLM but not shown to the user unless when used as a fallback `title`